zebra-scanner/src/tests.rs

```diff
@@ -0,0 +1,346 @@
+//! Test that we can scan blocks.
+//!
```

Comment:
Incomplete comment, or extra trailing empty line.

Optional: It could be helpful to summarise the tests and how they work here. 1-2 sentences is enough.